src/goto-symex/renaming_level.h

```diff
@@ -38,10 +38,9 @@ struct symex_renaming_levelt
   }
 
   /// Increase the counter corresponding to an identifier
-  void increase_counter(const irep_idt &identifier)
+  static void increase_counter(const current_namest::iterator &it)
```

Comment:
:pick: Would have been better to create two different commits for these two changes

Reply:
I'm not sure which two changes you are talking about